exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/customize.ipynb

```diff
@@ -0,0 +1,300 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e9720186-a4b2-43f3-81d5-3f96adc48e60",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "show_files([slctmanager.slc_dir.custom_pip_file])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "be4ac7a0-b7de-4168-b177-9b728f4d9a30",
+   "metadata": {},
+   "source": [
+    "We are now going to append the \"xgboost\" Python package to one of the package lists by adding `xgboost|2.0.3` and `scikit-learn|1.5.0` to the `flavor_customization/packages/python3_pip_packages` file. \n",
+    "Notes:\n",
+    " - running the following command multiple times will iteratively append the packages\n",
+    " - you can also click on the link and modify the file directly"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6b1ebb00-3ba2-4bbc-b923-f9a7d66d5a1f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xgboost_pkg = slct_manager.PipPackageDefinition(pkg=\"xgboost\", version=\"2.0.3\")\n",
+    "scikit_learn_pkg = slct_manager.PipPackageDefinition(pkg=\"scikit-learn\", version=\"1.5.0\")\n",
+    "slctmanager.append_custom_packages([xgboost_pkg, scikit_learn_pkg])\n",
+    "show_files([slctmanager.slc_dir.custom_pip_file])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a841d351-2bc3-49c1-89e3-d84c265fbf69",
+   "metadata": {},
+   "source": [
+    "#### Rebuilding the customized Flavor\n",
+    "\n",
+    "After changing the flavor you need to rebuild it. You can do it by running `export` again. Exaslct automatically recognizes that the flavor has changed and builds a new version of the container. Don't get confused by the warnings: `exaslct` first tries to find the cached docker images (see [Advanced Topics](./advanced.ipynb)), but as the content has changed, the cached image is not available, and the docker service returns a 404 error message."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "06e8ee1f-7121-4e78-8a7f-35951bf685df",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slctmanager.export()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c58b9172-e607-437a-9568-4063a88d446f",
+   "metadata": {},
+   "source": [
+    "Lets check the resulting tar gz file:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aabe7634-6d47-40c8-a363-4f53bf8702f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "show_directory_content(slctmanager.working_path.export_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc792fc1-5591-4449-879e-8c2866e2a21c",
+   "metadata": {},
+   "source": [
+    "#### Upload the Container to the Database\n",
+    "To use the new container you need to upload it to the BucketFS. If the build machine has access to the BucketFS you do it with the `exaslct` upload command, as shown below. Otherwise you need to install the script-languages-container manually: \n",
```

Comment:
```suggestion
    "To use the new container you need to upload it to the Bucket File System (BucketFS) of the database. If the build machine has access to the BucketFS you can use command `exaslct upload` as shown below. Otherwise you need to install the script-languages-container manually: \n",
```